packages/api/endpoints/s3credentials-readme/index.js

```diff
@@ -0,0 +1,17 @@
+const template = require('lodash/template');
+const { inTestMode } = require('@cumulus/common/test-utils');
+const instructions = inTestMode() ? `<a href="${process.env.DISTRIBUTION_ENDPOINT}s3credentials" target="_blank">${process.env.DISTRIBUTION_ENDPOINT}s3credentials</a>` : require('./instructions/index.html');
```

Comment:
`ava` was not liking the html import here, so this was my workaround. Tried a lot of other things to get it working. Open to suggestions